WooCommerce/WooCommerceTests/ViewRelated/Products/Edit Product/ProductFormViewModel+ObservablesTests.swift

```diff
@@ -201,14 +201,17 @@ final class ProductFormViewModel_ObservablesTests: XCTestCase {
         XCTAssertEqual(updatedUpdateEnabled, false)
     }
 
-    func testObservablesFromUploadingAnImage() {
-        // Arrange
+    func test_observables_from_mockProductImageUploader_when_uploadMediaAssetToSiteMediaLibrary_then_returns_true() {
+        // Given
         let product = Product.fake()
```

Comment:
Don't we have all this in` setUp() `? : )

Reply:
This specific class doesn't have a `setUp()`, only a `tearDown()` method (I'm not sure why). I left it as it is for now so the PR didn't become even bigger, but I opened a new issue here to refactor these here: https://github.com/woocommerce/woocommerce-ios/issues/7693